system-int-test/test/cmr/system_int_test/search/service/service_search_test.clj

```diff
@@ -219,7 +220,31 @@
       ;; Combination of params
       "Combination of params"
       [service3]
-      {:native-id "svc*" :provider "PROV2" "options[native-id][pattern]" true})))
+      {:native-id "svc*" :provider "PROV2" "options[native-id][pattern]" true}
+
+      "By service type"
+      [service1 service2 service3]
+      {:type "OPeNDAP"}
+
+      "By service type case insensitive by default."
+      [service1 service2 service3]
+      {:type "OPeNDAp"}
+
+      "By service types"
+      [service1 service2 service3 service4]
+      {:type ["OPeNDAP" "Harmony"]}
+
+      "By service type pattern true"
+      [service1 service2 service3]
+      {:type "OPeN*" "options[type][pattern]" true}
+
+      "By service type pattern false"
+      []
+      {:type "OPeN*" "options[type][pattern]" false}
+
+      "By service type ignore case true"
+      [service1 service2 service3]
+      {:type "OPeNDAp" "options[type][ignore-case]" true})))
```

Comment:
The tests should show searching with something like "opendap" found service1, service2 and service3 when ignore_case is `true` or ignore_case is missing (the default case) and found nothing when ignore_case is `false`.

Reply:
I have added tests using all lower case opendap, The last test case above I used OPeNDAp where the last character is lower case, but the actual value has an upper case P, so it is really covering the same test, but it is harder to see.  I added testing with ignore-case false when opendap is lower case and mixed case.